hunts/models.py

```diff
@@ -133,3 +137,59 @@ def get_minutes_per_solve(self, recent=False):
 
         minutes_elapsed = (interval_end - interval_start).total_seconds() / 60
         return "{:.2f}".format(round(minutes_elapsed / solved, 2))
+
+
+class HuntSettings(models.Model):
+    hunt = models.OneToOneField(
+        Hunt,
+        on_delete=models.CASCADE,
+        related_name="settings",
+    )
+
+    #
+    # Google-specific settings
+    #
+
+    # The id of your Google Drive folder
+    # This should be part of the URL (https://drive.google.com/drive/folders/<folder_id>)
+    google_drive_folder_id = models.CharField(max_length=128, blank=True)
+
+    # The id of your Google Sheets template file
+    # This should be part of the url (https://docs.google.com/spreadsheets/d/<sheet_id>)
+    google_sheets_template_file_id = models.CharField(max_length=128, blank=True)
+
+    # A link to the Google Drive folder that solvers can use for misc. files
+    google_drive_human_url = models.URLField(blank=True)
+
+    #
+    # Discord-specific settings
+    #
+
+    # The id of your Discord server
+    # This can be found on the "Widget" page in the Server Settings
+    discord_guild_id = models.CharField(max_length=128, blank=True)
+
+    # The id of the Discord channel to make puzzle announcements in
+    # This channel can get noisy and is recommended to be its own separate channel
+    # This ID can be found by enabling Developer Mode in Discord and right-clicking on the channel
+    discord_puzzle_announcements_channel_id = models.CharField(
+        max_length=128, blank=True
+    )
+
+    # The category name to create all Discord text channels in
+    discord_text_category = models.CharField(
```

Comment:
We will not need `discord_text_category` and `discord_voice_category` anymore after we switch over to organizing channels under their meta, right?

Reply:
Right, these fields might end up being pretty temporary.